crates/client/db/src/class_db.rs

```diff
@@ -87,19 +87,19 @@ impl MadaraBackend {
         Ok(self.db.get_pinned_cf(&col, &key_encoded)?.is_some())
     }
 
-    #[tracing::instrument(skip(self, id, class_hash), fields(module = "ClassDB"))]
+    #[tracing::instrument(skip(self, id, compiled_class_hash), fields(module = "ClassDB"))]
     pub fn get_sierra_compiled(
```

Comment:
get_sierra_compiled using the compiled class hash instead of the class hash breaks blockifier_state_adapter right? or was it already using compiled class hash and this field was just misnamed

Reply:
It’s just a misnaming that cost me time😅
but we should map class_hash directly to compiled_class for faster access and to avoid loading class_info beforehand

Reply:
ooh yes
but i think that blockifier state adapter is dumb and just gives us a compiled class hash, and that was the motivation for it? i dont remember